ios/CDVBackgroundGeolocation/CDVBackgroundGeolocation.m

```diff
@@ -496,9 +496,59 @@ -(void) onFinishLaunching:(NSNotification *)notification
             [facade start:nil];
             [facade switchMode:MAURBackgroundMode];
         }
+
+        UNUserNotificationCenter *center = [UNUserNotificationCenter currentNotificationCenter];
+
+        // Launched in background likely after app terminate, display notification that background geolocation is restricted.
+        if (@available(iOS 10, *))
+        {
+            [self isNotificationPermitted:center withCompletionHandler:^(BOOL permitted) {
+                if (permitted) {
+                    [self scheduleNotification:center];
+                }
+            }];
+        }
     }
 }
 
+-(void)scheduleNotification:(UNUserNotificationCenter *)center
+{
+    UNMutableNotificationContent *content = [[UNMutableNotificationContent alloc] init];
+    content.title = @"SiteSense - Background Geolocation Notification";
+    content.body = @"App has been terminated due to out of memory or closed by user. Background BLE scanning is not active or active with restricted performance. Tap here to wake up the app and resume normal scanning.";
```

Comment:
If we wanted to keep this in the spirit of the original code, especially if we wanted to provide any PRs back to them, the notification text would likely need to be configurable/passed in by the calling app.